makejdk.sh

```diff
@@ -260,12 +260,9 @@ buildAndTestOpenJDKViaDocker()
 
 testOpenJDKInNativeEnvironmentIfExpected()
 {
-  if [[ ! -z $JTREG ]]; then
-    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+  if [[ "${JTREG}" == "true" ]];
+  then
```

Comment:
So we agree that all if block conditions will be represented as for all our scripts:  
`if [[ ${...} == "...." ]];`